workspaces/optic-engine-wasm/src/lib.rs

```diff
@@ -380,6 +380,28 @@ pub fn spec_resolve_response(
     .map_err(|err| JsValue::from(format!("responses could not be serialized: {:?}", err)))
 }
 
+#[wasm_bindgen]
+pub fn spec_endpoint_delete_commands(
+  spec: &WasmSpecProjection,
+  path_id: String,
+  method: String,
+) -> Result<String, JsValue> {
```

Comment:
This is the main interface for Spectacle to integrate with. Thanks to the `wasm_bindgen` annotation this results in `import { spec_endpoint_delete_commands } from '@useoptic/optic-engine-wasm` to become functional.